data/gloperate/qml/SettingsPage.qml

```diff
@@ -54,6 +54,25 @@ Item
                 {
                     Layout.alignment: Qt.AlignRight
 
+                    text: 'Log Level'
+                }
+
+                ComboBox
+                {
+                    model: [ 'Critical', 'Error', 'Warning', 'Message', 'Debug', 'Debug (2)', 'Debug (3)', 'Debug (4)' ]
```

Comment:
Yes. Or we choose a different control (spinner or slider), because in the end we are selecting an unsigned integer here.